airbyte-integrations/connectors/source-netsuite/source_netsuite/source.py

```diff
@@ -47,30 +47,13 @@ def check_connection(self, logger, config: Mapping[str, Any]) -> Tuple[bool, Any
         object_types = config.get("object_types")
         base_url = self.base_url(config)
         session = self.get_session(auth)
-        # if record types are specified make sure they are valid
```

Comment:
Without re-writing this logic, removed connectivity check for each specific object.